packages/core/src/checkbox/checkbox.scss

```diff
@@ -1,16 +1,18 @@
+$variants: default, valid, invalid;
+$variant-selectors: ':not([valid],[invalid])', ':where([valid])', ':where([invalid])';
+$states: unchecked, checked;
+$state-selectors: '', ':where(:--checked, :--indeterminate, [state--checked], [state--indeterminate])';
+$sizes: md, lg;
+
 :host {
-  --_background: #fff;
+  --_border-radius: var(--sl-border-radius-checkbox);
+  --_border-size: var(--sl-border-width-input-border);
   --_border: var(--_border-size) solid var(--_border-color);
-  --_border-color: #ccc;
-  --_border-radius: 3px;
-  --_border-size: 1px;
-  --_gap: 0.25rem;
-  --_line-height: 1.5rem;
-  --_path-length: 22.910259;
-  --_size: calc(1rem - var(--_border-size) * 2);
-  --_stroke: #fff;
+  --_line-height: var(--sl-text-typeset-line-height-md);
+  --_path-length: 23;
   --_stroke-dashoffset: var(--_path-length);
   --_transition: 200ms ease-in-out;
+  --_icon-size: 16px;
```

Comment:
This should come from a token (eventually), right?

Reply:
yes, but it's not available yet. we decided that based on how far/definite the icon component is we'll fix this in a separate story so this can be merged